index.html

```diff
@@ -2236,6 +2236,10 @@ <h2>Service Endpoints</h2>
       <pre class="example nohighlight" title="Various service endpoints">
 {
   "service": [{
+    "id":"did:example:123#linked-domain",
+    "type": "LinkedDomains",
+    "serviceEndpoint": "https://bar.example.com"
```

Comment:
Interested in hearing what @jandrieu has to say about this.

`service` feels like a footgun feature... but one can't deny that it also has a variety of useful features. I'm still not clear on group consensus wrt. when it's fine to put in things that would cause trouble from a GDPR/PII perspective and when it's fine.

For example, if people take issue with `"type": "Person"`, then they should probably also have a huge problem with the service endpoint above?